src/upload-button.tsx

```diff
@@ -77,66 +83,85 @@ const FileConflictDialog = ({
 
     return (
         <Modal
-          position="top"
-          // @ts-expect-error incorrect PatternFly typing https://github.com/patternfly/patternfly-react/issues/10361
-          title={fmt_to_fragments(_("Replace file $0?"), <b>{uploadFile.name}</b>)}
-          titleIconVariant="warning"
-          variant={ModalVariant.medium}
-          onClose={handleCancel}
-          isOpen
-          footer={
-              <>
-                  <Button variant="warning" onClick={handleReplace}>{_("Replace")}</Button>
-                  {isMultiUpload &&
-                  <Button variant="secondary" onClick={handleSkip}>{_("Keep original")}</Button>}
-                  <Button variant="link" onClick={handleCancel}>{_("Cancel")}</Button>
-              </>
-          }
+            position="top"
+            // @ts-expect-error incorrect PatternFly typing https://github.com/patternfly/patternfly-react/issues/10361
+            title={fmt_to_fragments(
+                _("Replace file $0?"),
+                <b>{uploadFile.name}</b>,
+            )}
+            titleIconVariant="warning"
+            variant={ModalVariant.medium}
+            onClose={handleCancel}
+            isOpen
+            footer={
+                <>
+                    <Button variant="warning" onClick={handleReplace}>
+                        {_("Replace")}
+                    </Button>
+                    {isMultiUpload && (
+                        <Button variant="secondary" onClick={handleSkip}>
+                            {_("Keep original")}
+                        </Button>
+                    )}
+                    <Button variant="link" onClick={handleCancel}>
+                        {_("Cancel")}
+                    </Button>
+                </>
+            }
         >
             <p>
                 {cockpit.format(
-                    _("A file with the same name already exists in \"$0\". Replacing it will overwrite its content."),
-                    path.join('/')
+                    _(
+                        'A file with the same name already exists in "$0". Replacing it will overwrite its content.',
```

Comment:
Ah found a concrete example of the gettext issue. I'm not sure about removing the `\` from `\"$0\"` as well.

Reply:
Yes, this is absolutely invalid. Translatable strings **have** to use double-quotes to get recognized by xgettext. `test/static-code` also enforces this, but either it's not running or (most probably) it doesn't see this due to the line break.